indico/modules/events/timetable/templates/pdf/_program.html

```diff
@@ -0,0 +1,339 @@
+            <li class="inherited-border-color">
+                <div class="contrib-heading">
+                    {% if config.show_length_contribs and (not contrib.session or not contrib.session.is_poster) %}
+                        <span class="timebox inherited-bg-color inherited-txt-color">
+                            {{ (subcontrib.duration)|format_human_timedelta(narrow=True) }}
+                        </span>
+                    {% endif %}
+                    <h4>{{ subcontrib.title }}</h4>
+                </div>
+                <div class="contrib-timeline-content">
+                    {{ render_subcontrib(subcontrib, config) }}
+                </div>
+            </li>
+        {% endfor%}
+    {% endset %}
+
+    {% if contrib.session %}
+        <div>
+            <span class="bold">{% trans %}Subcontributions{% endtrans %}</span>
+            <ul class="sub-contrib-timeline inherited-bg-color-trans inherited-border-color">
+                {{ subcontribs }}
+            </ul>
+        </div>
+    {% else %}
+        <div class="session-block-content-box inherited-border-color">
+            <ul class="contrib-timeline">
+                {{ subcontribs }}
+            </ul>
+        </div>
+    {% endif %}
+{% endmacro %}
+
+{% macro render_subcontrib(subcontrib, config) %}
+    {{ render_child_contrib(subcontrib, config) }}
+{% endmacro %}
+
+{% macro render_entry_header(entry) %}
+    {% set object = entry.object %}
+    {% set sess = object.session %}
+    {% set type = entry.type.name %}
+
+    <h3>
+        {% if type == 'SESSION_BLOCK' %}
+            {{ render_block_title(object) }}
+        {% else %}
+            {{ object.title }}
+        {% endif %}
+    </h3>
+    <div class="session-block-title-info">
+        <p>
+            <span class="bold">
+                {% if type == 'BREAK' %}
+                    {% trans %}Break{% endtrans %}
+                {% elif type == 'CONTRIBUTION' %}
+                    {% trans %}Contribution{% endtrans %}
+                {% elif sess.is_poster %}
+                    {% trans %}Poster Session{% endtrans %}
+                {% else %}
+                    {% trans %}Session{% endtrans %}
+                {% endif %}
+            </span>
+        </p>
+        <p>
+            <span class="bold">{% trans %}Location{% endtrans %}:</span>
+            {{ render_place(entry) }}
+        </p>
+        {% if type != 'BREAK' %}
+            {% set people = object.person_links if type == 'SESSION_BLOCK' else object.speakers %}
+            <span class="bold">
+                {% if type == 'SESSION_BLOCK' %}
+                    {{ ngettext('Convener:', 'Conveners:', people|length) }}
+                {% else %}
+                    {{ ngettext('Speaker:', 'Speakers:', people|length) }}
+                {% endif %}
+            </span>
+            {{ render_names(people, show_title=config.show_title, show_affiliation=config.show_affiliation) }}
```

Comment:
Quite nitpicky but if you look at the PDF, there's now no space between `:` and the speaker/convener names. Maybe we could use something like `+ ' '` or `&nbsp;`

![Screenshot from 2024-11-15 09-43-17](https://github.com/user-attachments/assets/9a592269-6e75-4ec1-8207-34c6439d73cd)